ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp

```diff
@@ -1018,7 +1020,7 @@ template <typename unary_operation_t>
 void bind_unary_composite_float_with_default(py::module& module, const unary_operation_t& operation, const std::string& parameter_name_a, const std::string& parameter_a_doc, float parameter_a_value, const std::string& description) {
     auto doc = fmt::format(
```

Comment:
Change the description to note/info (to match the naming with the param we are passing).